examples/component/constraint/JointConstraint/JointConstraint.py

```diff
@@ -0,0 +1,71 @@
+import os
+from math import pi
+
+dirPath = os.path.dirname(os.path.abspath(__file__)) + '/'
+
+
+def createScene(rootNode):
+    rootNode.addObject('RequiredPlugin', name='SoftRobots')
+    rootNode.addObject('RequiredPlugin', name='SoftRobots.Inverse')
```

Comment:
@EulalieCoevoet Could you make sure that this plugin is not required? Thanks

Reply:
Oups sorry. Not needed indeed.